app/src/views/public/public-view.vue

```diff
@@ -74,14 +74,6 @@ import { storeToRefs } from 'pinia';
 import { computed } from 'vue';
 import { useI18n } from 'vue-i18n';
 
-interface Props {
-	wide?: boolean;
-}
```

Comment:
Luckily for me, this wasn't used in the first place. Refactoring private-view will be another story though